.env.example

```diff
@@ -9,4 +9,4 @@ SITE_URL=http://localhost:8000
 EMAIL_URL=dummymail://
 DEFAULT_FROM_EMAIL=webmaster@localhost
 SERVER_EMAIL=root@localhost
-ADMINS=Root <root@localhost>
+ADMINS=Root User <root@localhost.dev>
```

Comment:
I should probably create an issue for the "broken" email regex in the library...

Reply:
https://github.com/web-push-libs/vapid/pull/90